plugins/TranslationServer.js

```diff
@@ -270,7 +270,9 @@ var postHandler = function(data) {
     // loadMapFromString arguments: map, XML, preserve ID's, hoot:status
     hoot.loadMapFromString(map, data.osm, true);
     translation.apply(map);
```

Comment:
This keeps the JSON coming back inside the XML tags for the TranslationServer tests intact.